### PR TITLE
Add things to tracking entity and such.

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

**Comment by hedvigoscar** [comment]
This code was less complete than I thought. Do not approve. Do not merge. 🙁

**Comment by hedvigoscar** [comment]
Now it's a bit better @HedvigInsurance/clients

File: src/resolvers/createSession.ts [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+    campaignPromise = registerCampaign(token, headers, {
+      utmSource: campaign.source,
+      utmMedium: campaign.medium,
+      utmContent: campaign.content ? [campaign.content] : undefined,
+      utmCampaign: campaign.name,
+      utmTerm: campaign.term ? [campaign.term] : undefined,
+    })
+  }
+
+  let trackingIdPromise
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/resolvers/createSession.ts]: What I really wanted to express was that block of code, actually! :+1:

File: src/api/index.ts [hunk 2]
```diff
@@ -287,6 +295,34 @@ const registerDirectDebit = async (
   return data.json()
 }
 
+const registerCampaign = async (
```
Comment (palmfjord) [inline src/api/index.ts]: no need for this to be async anymore

Comment (hedvigoscar) [reply src/api/index.ts]:
```suggestion
const registerCampaign = (
```

File: src/api/index.ts [hunk 3]
```diff
@@ -287,6 +295,34 @@ const registerDirectDebit = async (
...
+  callApi('/hedvig/register_campaign', {
+    mergeOptions: {
+      headers: (headers as any) as RequestInit['headers'],
+      method: 'POST',
+      body: JSON.stringify(body),
+    },
+    token,
+  })
+
+const assignTrackingId = async (
```
Comment (hedvigoscar) [inline src/api/index.ts]:
```suggestion
const assignTrackingId = (
```

**Pull request merged**